podman/domain/containers.py

```diff
@@ -166,10 +168,13 @@ def exec_run(
             demux: Return stdout and stderr separately
```

Comment:
please, add a docstring for stream
```
demux (bool): Return stdout and stderr separately
stream (bool): Stream response data. Ignored if ``detach`` is true.
    Default: False
```

Reply:
@MattBelle don't forget the docstring, please

Reply:
Just a docstring update isn't enough. `deatch` + `stream` as is causes a buffer error because the `header` doesn't have enough bytes to unpack. Will add logic to to skip the frame streaming if `detach` is set.

Reply:
Detach should be working as described now. Added a test for it.